### DISCUSSION TITLE
How to watch for new discussion posts

### DISCUSSION THREAD

**Discussion by joe94** [community/community#4141] [Discussions] [answered]
Unless I missed some settings somewhere, I have not been able to find a way to get notifications for new posts. I get notifications only once I start participating in one. These notifications would be a good way for my team to know when someone posted something new.

**Comment by ajsirota** [comment, marked as answer]
Click "Watch" at the top of the page, then choose "All Activity" to be notified for all activity, or choose "Discussions" under "Custom" to only subscribe to notifications for discussions.

**Comment by Feri891** [comment]
Ok
